components/net/mime_classifier.rs

```diff
@@ -436,8 +438,8 @@ impl BinaryOrPlaintextClassifier {
         } else if data.iter().any(|&x| {
             x <= 0x08u8 ||
                 x == 0x0Bu8 ||
-                (x >= 0x0Eu8 && x <= 0x1Au8) ||
-                (x >= 0x1Cu8 && x <= 0x1Fu8)
+                (0x0Eu8..=0x1Au8).contains(&x) ||
+                (0x1Cu8..=0x1Fu8).contains(&x)
```

Comment:
There is now an off by one error in the `contains(...)` call as Rust ranges are exclusive on the end value.

Reply:
I think the `start..=end` syntax represents an [inclusive range](https://doc.rust-lang.org/std/ops/struct.RangeInclusive.html), but I may be misinterpreting it.

Reply:
Oh, fascinating. I didn't know about that syntax!